### ISSUE TITLE
Alias does not preserve schema

### ISSUE THREAD

**Issue by Karel-van-de-Plassche** [coleifer/peewee#1518]
Using peewee 3.1.0. I'm trying to use an alias in a self-join to create a common table expression. However, I get an error that the aliased table name does not exists. I think this is because alias does not preserve the schema. Minimal example:

``` Python
netAlias = Network.alias('net')
cte = Network.select(Network.id, netAlias.networks).from_(Network, netAlias).cte('nets')
query = net.select().with_cte(cte)
query.sql()

('WITH "nets" AS (SELECT "t1"."id", "net"."networks" FROM "develop"."network" AS "t1", "network" AS "net") SELECT "t2"."id", "t2"."feature_names", "t2"."target_names", "t2"."recipe", "t2"."networks" FROM "develop"."network" AS "t2"',
 [])

query.get()

ProgrammingError: relation "network" does not exist
LINE 1: ...net"."networks" FROM "develop"."network" AS "t1", "network" ...
```

I would expect from_ in the cte to generate `"develop"."network" AS "net"`, not `"network" as "net"`. However, I'm an SQL noob and might be using cte's wrong.

**Comment by Karel-van-de-Plassche** [comment]
Maybe related `netAlias.select().get()` fails too, although `netAlias.get()` works.

**Comment by coleifer** [comment]
Fixed in d7f0350, thanks so much for reporting.
